pulse2percept/models/base.py

```diff
@@ -417,6 +418,53 @@ def plot(self, use_dva=False, style='hull', autoscale=True, ax=None,
             ax.set_ylabel('y (microns)')
         return ax
 
+    def __eq__(self, other):
+        """
+        Equality operator for Spatial Model.
+        Compares two Spatial Models based attribute equality
+
+        Parameters
+        ----------
+        other: SpatialModel
+            SpatialModel to compare with
+
+        Returns
+        -------
+        bool:
+            True if the compared objects have identical attributes, False otherwise.
+        """
+        if not isinstance(other, SpatialModel):
```

Comment:
This looks like the only line not shared between `SpatialModel` and `TemporalModel` `__eq__`. Checking to see if other is an instance of `self.__class__` might work to unify the two functions in `BaseModel`